R/database.R

```diff
@@ -1430,40 +1488,59 @@ get_ssm_by_regions = function(regions_list,
   if(missing(maf_data)){
     region_mafs = lapply(regions, function(x){get_ssm_by_region(region = x,
                                                                 streamlined = streamlined,
+                                                                basic_columns = TRUE,
                                                                 from_indexed_flatfile = from_indexed_flatfile,
                                                                 mode = mode,
                                                                 augmented = augmented,
                                                                 seq_type = seq_type,
-                                                                projection = projection,
-                                                                allow_clustered = allow_clustered)})
+                                                                projection = projection)})
   }else{
     region_mafs = lapply(regions, function(x){get_ssm_by_region(region = x,
                                                                 streamlined = streamlined,
                                                                 maf_data = maf_data,
                                                                 from_indexed_flatfile = from_indexed_flatfile,
-                                                                mode = mode,
-                                                                allow_clustered = allow_clustered)})
+                                                                mode = mode)})
   }
+
   if(!use_name_column){
     rn = regions
   }else{
     rn = regions_bed[["name"]]
   }
+
+  #get region names
   tibbled_data = tibble(region_mafs, region_name = rn)
   unnested_df = tibbled_data %>%
     unnest_longer(region_mafs)
 
+  #subset on region anmes and rename column before cbind with full maf
+  r_names = as.data.frame(unnested_df$region_name)
+  colnames(r_names)[1] <- "Region_Name"
```

Comment:
Here, this line is related to the comment above. What is the default name of first column here?

Reply:
The renaming of the column is necessary since the unnesting step gives it a different name (dfname$columnname). The renaming was reverted to "region_name" to match what's expected downstream and the redundant renaming step was removed (in get_ashm_count_matrix).